src/GrooveApiDataContract/ErrorCode.cs

```diff
@@ -123,27 +123,27 @@ public enum ErrorCode
         #endregion
 
         #region AccessToken errors
-        [Description("Azure Marketplace access token required")]
+        [Description("Microsoft access token required")]
```

Comment:
>Microsoft [](start = 22, length = 9)

I'm not sure that Microsoft access token or Microsoft token are things which exist. We should use the same words as those used in the documentation for consistency. ("application access token" & "user access token" I guess)